grafonnet/stat_panel.libsonnet

```diff
@@ -74,7 +74,7 @@
     links: [],
     [if repeat != null then 'repeat']: repeat,
     [if repeat != null then 'repeatDirection']: repeatDirection,
-    [if repeat != null then 'repeatMaxPerRow']: repeatMaxPerRow,
+    [if repeat != null then 'maxPerRow']: repeatMaxPerRow,
```

Comment:
Seems to me we should name the argument `maxPerRow` as well. Would you mind updating that and the documentation comment as well?

Reply:
Okay, I'll update that.